pkgs/applications/video/obs-studio/obs-move-transition.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, fetchurl
+, cmake
+, obs-studio
+}:
+
+let
+  findlibobs = fetchurl {
+    url = "https://raw.githubusercontent.com/obsproject/obs-studio/master/cmake/external/FindLibobs.cmake";
+    sha256 = "1x1kkkrxwrvr682f687xvfa8jpi10y873c7xgcbz9v4nbj5hp4wb";
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "obs-move-transition";
+  version = "2.0.2";
+
+  src = fetchFromGitHub {
+    owner = "exeldro";
+    repo = "obs-move-transition";
+    rev = "42b3e46f73d23929796a11c3c29fa294212d045d";
```

Comment:
Is there a reason why you did not track a tag?

Reply:
The author of the project doesn't seem to tag his releases anymore. The commit I locked bumped the release to 2.0.2 and the author states that the latest release is 2.0.2 on the [OBS plugin page](https://obsproject.com/forum/resources/move-transition.913/).

Reply:
Can you please open an issue upstream to request tags? The PR is not blocked on this, but this would be nice to have to get automated upgrades.

Reply:
Issue opened: https://github.com/exeldro/obs-move-transition/issues/52